app/services/notifications/notifiable_action/send.rb

```diff
@@ -16,16 +16,16 @@ def self.call(*args)
       end
 
       def call
+        return unless notifiable.is_a?(Article)
```

Comment:
This entire service makes the assumption that `notifiable` will be an `Article` and calls methods on it assuming that it will _always_ be an `Article`. However, there aren't any safeguards for this, which is why I added this line.